rspec-support.gemspec

```diff
@@ -33,8 +33,8 @@ Gem::Specification.new do |spec|
     spec.cert_chain = [File.expand_path('~/.gem/rspec-gem-public_cert.pem')]
   end
 
-  spec.required_ruby_version = '>= 1.8.7'
+  spec.required_ruby_version = '>= 2.3.0'
 
-  spec.add_development_dependency "rake",         "> 10.0.0"
+  spec.add_development_dependency "rake", ">= 12.3.3"
```

Comment:
I preferred to avoid updating libraries versions, [`12.3.3` comes from the `Gemfile`](https://github.com/rspec/rspec-support/blob/main/Gemfile#L21):
```ruby
gem 'rake', '>= 12.3.3'
```